### PR TITLE
deps: Bump `com_google_googleapis` -> 82c0f26d

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -255,8 +255,9 @@ def envoy_dependencies(skip_targets = []):
         cc = True,
         go = True,
         grpc = True,
+        python = True,
```
Comment (phlax) [inline bazel/repositories.bzl:258]: not sure if this is necessary...

**Review by keith — APPROVED** [review]

**Comment by phlax** [comment]
argg

```console
ERROR: Evaluation of subquery "deps(set(//test/...))" failed (did you want to use --keep_going?): preloading transitive closure failed: no such target '@com_google_googleapis//google/api/expr/v1alpha1:syntax_py_proto_genproto': target 'syntax_py_proto_genproto' not declared in package 'google/api/expr/v1alpha1' defined by /build/tmp/_bazel_envoybuild/b570b5ccd0454dc9af9f65ab1833764d/external/com_google_googleapis/google/api/expr/v1alpha1/BUILD.bazel
```

**Comment by phlax** [comment]
/wait for protobuf dep to be updated - and then ill try this again

**Comment by phlax** [comment]
digging a bit further on this it seems that this commit https://github.com/googleapis/googleapis/commit/45b363d3ab4692115cc5018eb26dcdaa49b320a8 removes the `syntax_py_proto` that is used/referred to in `api_build_system`

not sure exactly why or how to resolve

**Comment by wbpcode** [comment]
/wait

Please fix ci first.

**Comment by phlax** [comment]
closing in favour of #24914

**Pull request closed without merging**